documentation.md

```diff
@@ -0,0 +1,14 @@
+# Documentation
+This guide aims to make it clear what information can and cannot be included in documents and designs that are made public.
```

Comment:
I think there is a question as to whether policy documentation should minimise references to the department and/or organisation and/or team names. I think we should discuss this with legal, and then record the rationale for whatever decision is made.

Reply:
I think it should minimise references, but only to make it more generic and reusable. The documents exist in the DataWorks team within the DWP org.